src/api.py

```diff
@@ -485,6 +485,14 @@ def firebase_token() -> ResponseType:
         return jsonify(ok=False)
 
 
+def _notify_opponent_about_move(opponent_id: str, move: MoveBase) -> None:
+    firebase.send_push_notification(
+        user_id=opponent_id,
+        title="Your opponent has made a move",
```

Comment:
There are 3 locales: 
 1. game locale
 2. user locale
 3. opponent locale
 Which one should I use when I send a notification? I thought that I should use opponent locale but I can't reach this field because we doesn't store it game model. Can I add it to game model? Or there is another way to do it?
 PS: I can't use localization on frontend because firebase sends notification directly to device using token